Makefile

```diff
@@ -35,8 +35,8 @@ RUN := run
 PKG          := ./...
 TAGS         :=
 TESTS        := ".*"
-TESTTIMEOUT  := 15s
-CPUS         := 1
+TESTTIMEOUT  := 1m
```

Comment:
OK, ratcheted down to 30s. It'd be great to get a list of slow tests, but that's not something you get from the standard test runner AFAIK.